konan/konan.properties

```diff
@@ -111,22 +111,22 @@ clangDebugFlags.ios_arm32 = -O0
 clangDynamicFlags.ios_arm32 =
 linkerNoDebugFlags.ios_arm32 = -S
 linkerDynamicFlags.ios_arm32 = -dylib
-linkerKonanFlags.ios_arm32 = -lSystem -lc++ -lobjc -framework Foundation -sdk_version 13.5
+linkerKonanFlags.ios_arm32 = -lSystem -lc++ -lobjc -framework Foundation -sdk_version 14.0
```

Comment:
Shouldn't we bump macOS SDK version too?

Reply:
macOS 11 is not released yet.

Reply:
Of course. But Xcode 12 includes SDK for macOS 10.15.6.